pycape/cape.py

```diff
@@ -92,7 +94,7 @@ def close(self):
         self._loop.run_until_complete(self._close())
         self._ctx = None
 
-    def connect(self, function_ref: Union[str, fref.FunctionRef]):
+    def connect(self, function_ref: Union[str, fref.FunctionRef], pcrs=None):
```

Comment:
Can you add typing for `pcrs` as you did  for `_key` and `bootstrap` ? thank you!